Project/Project/Models/Review.cs

```diff
@@ -14,6 +14,8 @@ public class Review
     public string Movie_id { get; set; }
     public string Title { get; set; }
     public string Text { get; set; }
+    public string movie_title { get; set; }
```

Comment:
To follow the standard please change to capitazed example Movie_title, Movie_poster